cdap-notifications/src/main/java/co/cask/cdap/notifications/service/kafka/KafkaNotificationService.java

```diff
@@ -79,8 +81,9 @@ public KafkaNotificationService(CConfiguration cConf, KafkaClient kafkaClient, D
     this.feedManager = feedManager;
     this.ack = KafkaPublisher.Ack.LEADER_RECEIVED;
     this.nbPartitions = cConf.getInt("kafka.num.partitions");
-
+    this.notificationTopic = cConf.get(Constants.Notification.KAFKA_TOPIC_PREFIX);
```

Comment:
Do Kafka topics allow arbitrary strings? Or do we need to validate that it is a valid topic name?